website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB data platform setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Crate.io, Inc.
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Not Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'Community Forum'
+  slack_channel_link: 'https://community.cratedb.com/'
+  platform_name: 'CrateDB'
+  config_page: '/reference/resource-configs/no-configs'
+---
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta}/>
+
+
+[CrateDB] is compatible with PostgreSQL, so its dbt adapter strongly depends on
+dbt-postgres, documented at [PostgreSQL profile setup].
+
+CrateDB targets are configured exactly the same way, see also [PostgreSQL
+configuration], with just a few things to consider which are special to
+CrateDB. Relevant details are outlined at [using dbt with CrateDB],
```

Comment:
This is a bit confusing because this link actually goes to no information. If there are no special configurations, I would just not even add this sentence for a better user experience.

Reply:
@mirnawong1 would welcome your thoughts here

Reply:
Found thread and comments. Will resolve now.